src/stack_aggregator.py

```diff
@@ -661,13 +662,16 @@ def execute(aggregated=None, persist=True):
                          'external_request_id': external_request_id,
                          'result': stack_data}
         # Ingestion of Unknown dependencies
-        logger.info("Unknown ingestion flow process initiated.")
-        try:
-            for dep in unknown_dep_list:
-                server_create_analysis(ecosystem, dep['name'], dep['version'], api_flow=True,
-                                       force=False, force_graph_sync=True)
-        except Exception as e:
-            logger.error('Ingestion has been failed for ' + dep['name'])
-            logger.error(e)
-            pass
+        if Settings().disable_unknown_package_flow:
+            logger.warning('Unknown package flow is disabled')
+        else:
```

Comment:
Instead of having else, do a early return if disable_unknown_package_flow is True